### PR TITLE
Show the full url of a file in the source tab tooltip (#4203)

### PR DESCRIPTION
**Pull request by ohana54**
Associated Issue: #4203 

### Summary of Changes

The source tab tooltip now shows the full file url instead of just the file name.

### Test Plan

- Open the debugger panel
- Open an example source file
- Tab titles should be the file name
- Hover over the tab - the tooltip should be the file url

### CODE REVIEW

**Comment by jasonLaster** [comment]
Thanks!

Looks good, perhaps some unit tests for the utils?

**Comment by ohana54** [comment]
Sorry about that, I'm on it

**Comment by ohana54** [comment]
Done.

File: src/utils/source.js [hunk 1]
```diff
@@ -101,12 +101,24 @@ function getRawSourceURL(url: string): string {
   return url.replace(/:formatted$/, "");
 }
 
-function getFilenameFromURL(url: string) {
+function resolveFileURL(
+  url: string,
+  transformUrl: string => string = initialUrl => initialUrl
```
[the remark this answers is not in the document]
Comment (ohana54) [reply src/utils/source.js]: Great idea! I'll do that later tonight.

[the remark this answers is not in the document]
Comment (ohana54) [reply src/utils/source.js]: Done.

**Pull request merged**